app/controllers/admin/resources_controller.rb

```diff
@@ -23,12 +23,25 @@ def create
     end
 
     def update
-      if @resource.update_attributes(resource_params)
-        redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
-                    notice: 'Resource successfully updated.'
+      if params[:inc].present?
+        @resource.used = params[:inc].to_i == 1 ? (@resource.used - 1) : (@resource.used + 1)
+        flag = @resource.save
       else
-        flash[:error] = "Resource update failed: #{@resource.errors.full_messages.join('. ')}."
-        render :edit
+        flag = @resource.update_attributes(resource_params)
+      end
+      respond_to do |format|
+        if flag
+          format.html do
+            redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
+                        notice: 'Resource successfully updated.'
+          end
+          flash[:notice] = "One #{params[:inc].to_i == 1 ? 'resource freed' : 'more resource used'}"
+          format.js{ render action: 'update_quantity' }
+        else
+          flash[:error] = "Resource update failed: #{@resource.errors.full_messages.join('. ')}."
```

Comment:
Again, I need to know which resource failed to update, otherwise this offers the organizer very little meaningful information.